packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -164,7 +164,7 @@ export default class MeetingInfoV2 {
             stack: err.stack
           }
         );
-        throw new MeetingInfoV2AdhocMeetingError(err.body?.code, err.body?.message);
+        throw err;
```

Comment:
I think we need this back

Reply:
Then we will need to throw the caught error directly this way on line number 156. Error thrown from webex.request's catch throws error in this format and from here again we format it making message like `<message>, <code>, <code>` i.e. code gets repeated twice. I think we should format at line 167 and throw directly from 156 to cover `conversation.get` call as well.

Please confirm.